lib/clover_force.cpp

```diff
@@ -44,22 +44,39 @@ namespace quda
 
     for (auto i = 0u; i < x.size(); i++) {
       gamma5(p[i][parity], x[i][parity]);
-
       if (dagger) dirac->Dagger(QUDA_DAG_YES);
       dirac->Dslash(x[i][other_parity], p[i][parity], other_parity);
       // want to apply \hat Q_{-} = \hat M_{+}^\dagger \gamma_5 to get Y_o
       dirac->M(p[i][parity], p[i][parity]); // this is the odd part of Y
       if (dagger) dirac->Dagger(QUDA_DAG_NO);
 
-      gamma5(x[i][other_parity], x[i][other_parity]);
-      if (detratio) blas::xpy(x0[i][parity], p[i][parity]);
+      if(inv_param.twist_flavor == QUDA_TWIST_NONDEG_DOUBLET){
+        // FIXME: here we used the already existing ca_lambda_max member of inv_param. Maybe it's better to create a new parameter for this purpose
```

Comment:
Yes, this should be a new parameter, `since ca_lambda_max` is for something else completely (it sets the maximum eigenvalue for communication avoiding CG when using a Chebyshev basis).

Reply:
Resolving this.  Not that whenever we add something to `QudaInvertParam` or `QudaGaugeParam` we need to mirror the addition in the Fortran interface (b32949e832ff7dd2c4c957a274cf8683b8202094).